ichnaea/async/tests.py

```diff
@@ -10,38 +9,85 @@
 
 class TestBeat(object):
 
-    def test_tasks(self, celery):
-        tmpdir = tempfile.mkdtemp()
-        filename = os.path.join(tmpdir, 'celerybeat-schedule')
+    def test_tasks(self, celery, tmpdir):
```

Comment:
This test was failing, so I spent some time tinkering with it and ended up tweaking it a bit.

Here, I switched it to use the pytest `tmpdir` fixture so we don't have to do our own tmpdir maintenance work.